src/models/eventmodel.cpp

```diff
@@ -39,7 +41,7 @@ quintptr combineDataTag(Tag tag, quintptr data)
 Tag dataTag(quintptr internalId)
 {
     auto ret = (internalId << DATATAG_UNSHIFT) >> DATATAG_UNSHIFT;
-    if (ret > static_cast<quintptr>(Tag::Threads))
+    if (ret > static_cast<quintptr>(Tag::LAST_TAG))
```

Comment:
this is wrong in your code, will get correct with my code (your patch as-is would need `>=`)

just saying, keep this as-is - just change `LAST_TAG` as indicated above